programmers/문자열-내-p와-y의-개수.js

```diff
@@ -0,0 +1,17 @@
+function solution(s) {
+  var answer = true;
+  let countP = 0;
+  let countY = 0;
+
+  for (let i in s) {
+    if (s[i] == "p" || s[i] == "P") {
+      countP += 1;
+    } else if (s[i] == "y" || s[i] == "Y") {
+      countY += 1;
+    }
+  }
+
+  answer = countP == countY ? true : false;
+
+  return answer;
```

Comment:
```suggestion
  return countP == countY ? true : false;
```
이런식으로도 사용이 가능하니 참고해주세요ㅎ

삼항 연산자 사용을 알고계신다니 놀랍습니다...